prac_05/state_names.py

```diff
@@ -0,0 +1,34 @@
+"""
+CP1404 2020 - Practical 5
+Student Name: Amy Robinson
+Program - State Names
+
+State names in a dictionary
+File needs reformatting
+"""
+
+# Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales",
+                "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory",
+                "VIC": "Victoria", "TAS": "Tasmania"}
+
+print(CODE_TO_NAME)
+
+state_code = input("Enter short state: ").upper()
+
+while state_code != "":
+    if state_code in CODE_TO_NAME:
+        print(state_code, "is", CODE_TO_NAME[state_code])
```

Comment:
Do you think that print statements should have consistent formats?

Reply:
Changed, Thank you.